client/cmdhfmf.c

```diff
@@ -1240,7 +1250,7 @@ int CmdHF14AMfChk(const char *Cmd)
 				PrintAndLog("Command execute timeout");
 			}
 		}
-	} else {
+	} else if (singleBlock) { // Ensure single block mode in case SectorsCnt == 0

```

Comment:
I still think that `SectorsCnt == 0` is never possible. See line 1100 ff above. This should be possible to simplify to `if (!singleBlock) { ... } else { ... }`

Reply:
Updated as requested.